server/migrations/20190510143954_alter_users_table.js

```diff
@@ -0,0 +1,10 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.table('users', function(table) {
+        table.string('resettoken').nullable();
```

Comment:
Not sure you can change this now, but next time please stick to conventions. All columns follow the convention of `_` separating words except yours.

Reply:
don't think it can be changed....will adhere to naming conventions from now on

Reply:
It can be changed. We'll change it later, when we don't have demo the next day.